### DISCUSSION TITLE
EcoCompute Conference 2024

### DISCUSSION THREAD

**Discussion by ArneTR** [Green-Software-Foundation/discussions#129] [Events]
Dive into the EcoCompute Conference on April 25th and 26th at the House of Communication in Munich. 🌿

A tech conference purely dedicated to sustainability in hardware & software! Join us in Munich on April 25-26 at the House of Communication. 

Immerse in cutting-edge talks, connect with industry leaders, explore practical solutions, attend workshops, network, connect and drive the topic of digital sustainability.

https://www.eco-compute.io/

**Comment by ArneTR** [comment]
@NAMRATA-WOKE @jawache Is anybody montoring this anymore?

I tried contacting the GSF over the last 4 weeks via E-Mail, Github and others to either get something listed in the Newsletter, get some Infos and Support for our Meetup Group which is part of the GSF Meetup Network or getting something accepted in the Awesome Software list.

No reply nowhere. What is going on?

**Reply by NAMRATA-WOKE** [reply]
Hi @ArneT, this week’s newsletter featured your Call for Speakers! We’ll continue to promote the event as the date approaches. We typically don't list it under events if it's more than a week away - 2 weeks if it's a big event.

**Reply by ArneTR** [reply]
Hey @NAMRATA-WOKE thanks for the quick reply, really appreciated!

Maybe we are speaking about a different newsletter? It arrived in my inbox this morning and no word about the call for speakers. Here: https://mailchi.mp/greensoftware/green-software-foundation-2024-mar-77

In any case: The call for speakers is closed since 40 days ... so featuring this would also be very confusing.

Thanks so for clarifying your policy about when you feature events, that helps! For a conference I think a week in advance is pretty late ... typically people make this decision months ahead, not?

2. Can you also tell me something about the other two questions: Who and where to contact about the Meetup Groups, and who to contact about the Awesome Software repository?

**Reply by NAMRATA-WOKE** [reply]
@ArneTR, my mistake - we shared the call for speakers for the Green IO event. If you need more runway, we can accommodate your request and share the event details sooner! 

I'll connect you to @Sophietn via email. She is the PM for Awesome Green Software. 

Regarding support for your Meetup group, can you provide more information about what you're looking for? We’ll do our best to help. We’re in the process of filling the role of someone who will lead our Meetup program, but we will do the best we can to support you in the meantime!

**Comment by ArneTR** [comment]
> @ArneTR, my mistake - we shared the call for speakers for the Green IO event. If you need more runway, we can accommodate your request and share the event details sooner!

@NAMRATA-WOKE That would be great! I hope all info is in this post. If you need more / other let me know.

2. Looking forward to the connect via email.

**Comment by Sophietn** [comment]
Hi @ArneTR 
Haven't seen an email come through, but I can answer some of your questions here. 

**Awesome Green Software** 
Apologies for the delay in reviewing your suggested submission to the database. 
We're about to start an election period to replace the Project Lead for the Awesome Green Software project, so there will continue to be a delay until the new lead is in position. We will be opening a nomination period for the elections shortly. 

**Meetup**
I haven't seen any requests from the [GreenCoding meetup group](https://www.meetup.com/green-coding/) for info or support. Please could you reshare? You can either contact directly using the messaging within the meetup platform, or contact our meetup mailing list (meetup@greensoftware.foundation). It'd be great to support you on an upcoming event soon!
